.env

```diff
@@ -6,10 +6,11 @@
 
 # Specify a Botkit Studio token so your bot can access cloud-hosted content and features
 # Get one here: https://studio.botkit.ai/
-#studio_token=''
+studio_token=''
 
-# Specify an APIAI token so your bot can access cloud-hosted content and features from Dialogflow
-apiai_token=''
+# Path to APIAI v2 keyfile so your bot can access cloud-hosted content and features from Dialogflow.
+# If you do not yet have a keyfile follow the instructions here to generate a service account key: https://dialogflow.com/docs/reference/v2-auth-setup 
+#apiai_keyfile=''
```

Comment:
is it right that this is commented out?

Reply:
it's basically one or the other, either botkit studio token, or the apiai keyfile.  it's up to the user to fill it out as needed.  so I think it's ok to have this commented out, which is better than having them both uncommented, which i think is more confusing.